index.html

```diff
@@ -5,32 +5,29 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <link rel="stylesheet" href="style.css">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My Portfolio</title>
 </head>
 <body>
-  <div class="wrapper">
+  <main class="wrapper">
     <div class="bar-nav">
-        <h1>My Logo</h1>
+        <a href="index.html" class="logo">My Logo</a>
         <img src="image/Union.png" alt="Menu icon">
```

Comment:
- [ ] (OPTIONAL) Since this hamburger image is the navigation for the mobile site, it would be great if you could wrap it in a `navbar` tag for best semantic practices 👍

Reply:
Done